### ISSUE TITLE
Teach R^2 instead of RMSPE?

### ISSUE THREAD

**Issue by trevorcampbell** [UBC-DSCI/introduction-to-datascience-python#131]
In `scikit-learn`, all the regressors use $R^2$ instead of RMSPE for evaluation. It makes using RMSPE kind of a pain -- we have to jump through hoops.

But it would be a pretty major change, so I'm just opening this for discussion once the textbook has stabilized. Certainly not a v1 thing to do.

**Comment by joelostblom** [comment]
There is some discussion on this here by Tiff and Mathias https://github.ubc.ca/UBC-DSCI/dsci-100-instructor/issues/43. I personally find the name RMS**P**E a bit confusing since the formula is the same as RMSE and it is just the data that it is applied to that is different.

**Comment by joelostblom** [comment]
Just a note that after teaching in 2023 fall I came across this again and find it hard to justify to students (and myself) why we make a distinction in the name between RMSPE and RMSE whereas we don't do that for any other metric we use (none of accuracy, precision, recall don't change names when applied to test data) and if students look up other regression metrics to use with sklearn in the docs they will also not come across any difference in naming convention between the same metric applied to different data sets.

**Comment by trevorcampbell** [comment]
I think the RMSPE vs RMSE and R^2 vs RMS(P)E are different issues. Probably worth keeping the discussion here for now I guess because they're related, but might be worth spinning that into a separate issue at some point if it gets too disjoint.

On the RMSE vs RMSPE issue: I agree with you, but if we want to change that we'd need to discuss with the teaching team (and course design team) to make sure everyone is happy with it. It's also a fairly minor change compared to other stuff we're working on, so low priority.
